internal/controller/pkg/revision/runtime.go

```diff
@@ -194,6 +195,7 @@ func (b *RuntimeManifestBuilder) Deployment(serviceAccount string, overrides ...
 		// Optional defaults, will be used only if the runtime config does not
 		// specify them.
 		DeploymentWithOptionalName(b.revision.GetName()),
+		DeploymentWithLabels(b.podSelectors()),
```

Comment:
Not sure if adding it as an override is a good option. Other option is to add it in the runtime pre/post hook.

Reply:
I'm not sure either. @turkenh introduced this pattern, so I'm sure he can provide guidance.